doc/md/examples/grammar.txt

```diff
@@ -268,6 +271,10 @@
     'flexible'
     'stable'
 
+<stab_mod> ::= 
+    'flexible'
```

Comment:
If we don't like the option of

flexible actor (class...) 

we could just rule this production out...